lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -514,15 +534,41 @@ def post(self):
         are not in the class schema are ignored but logged.
         """
         json_data = request.get_json(silent=True)
+        if json_data and "user" in json_data:
```

Comment:
This assumes `json_data` is a `dict()`.  Is that always the case?

Reply:
Yeah I mean flask's `request.get_json` parses the data as JSON objects as key/value pairs, just like a Python dictionary. So its always going to be a dictionary I guess